### PR TITLE
cap_add support in k8s

### PR DESCRIPTION
**Pull request by chlgit**
Add capabilities for containers. For example, yelp-main main_uswest1 instances needs these capabilities: "DAC_OVERRIDE", "CHOWN", "SETGID", "SETUID"]

### CODE REVIEW

File: paasta_tools/kubernetes_tools.py [hunk 1]
```diff
@@ -553,6 +555,16 @@ def get_liveness_probe(
 
         return probe
 
+    def get_security_context(self):
```
Comment (vkhromov) [inline paasta_tools/kubernetes_tools.py]: please annotate the return type.

Comment (chlgit) [reply paasta_tools/kubernetes_tools.py]: kk, done.

File: paasta_tools/kubernetes_tools.py [hunk 2]
```diff
@@ -553,6 +555,16 @@ def get_liveness_probe(
 
         return probe
 
+    def get_security_context(self) -> V1SecurityContext:
```
Comment (vkhromov) [inline paasta_tools/kubernetes_tools.py]: since the function returns `None` in some cases, the return type is actually `Optional[V1SecurityContext]`.

Comment (chlgit) [reply paasta_tools/kubernetes_tools.py]: Ah, you're right. Let me fix it in the separated PR.

**Pull request merged**